packages/edit-navigation/src/components/menu-selector/style.scss

```diff
@@ -0,0 +1,16 @@
+.edit-navigation__menu-selector__select-menu-button {
+	width: 100%;
+}
+
+.edit-navigation-menu-selector {
+	padding: $grid-unit-15;
+	@extend .edit-navigation-editor;
```

Comment:
I'm not a sass expert, but I know `@extend` isn't commonly used. 

I think it's better to provide consistent styles through reusable components rather than mixing together css.

This shouldn't be an issue if we repurpose `EmptyState` as mentioned in an earlier comment. That uses a `Card` component to provide the right border style.